src/cgns_internals.c

```diff
@@ -81,7 +81,7 @@ void *cgi_malloc(size_t cnt, size_t size)
 {
     void *buf = calloc(cnt, size);
     if (buf == NULL) {
-        cgi_error("calloc failed for %d values of size %d", cnt, size);
+        cgi_error("calloc failed for %zu values of size %zu", cnt, size);
```

Comment:
This is good as long as we use C99 (or later) compiler.

Reply:
CGNS already requires C99+ to compile.